extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -193,10 +193,19 @@ public boolean isAnyPropertySet() {
         @ConfigItem
         public Optional<String> charset;
 
+        /**
+         * Whether Hibernate should quote all identifiers.
+         * <p>
+         * Default value is false.
```

Comment:
Can you remove the default value part of the java doc? We automatically generate the doc from the annotation.